app/presenters/filter_presenter.rb

```diff
@@ -0,0 +1,55 @@
+class FilterPresenter
+  def initialize(search_parameters, document_types, organisations)
+    @document_types = document_types
+    @search_parameters = search_parameters
+    @organisations = organisations
+  end
+
+  def document_type?
+    @search_parameters[:document_type].present?
+  end
+
+  def document_type_options
+    types = [{
+               text: 'All document types',
+               value: '',
+               selected: @search_parameters[:document_type] == ''
+             }]
+    @document_types.each do |document_type|
+      types.push(
+        text: document_type.try(:tr, '_', ' ').try(:capitalize),
+        value: document_type,
+        selected: document_type == @search_parameters[:document_type]
+      )
+    end
+    types
+  end
```

Comment:
Small thing - could we re-shuffle `document_type_options` and `organisation_options` to be next to each other?